.github/workflows/dlp-pipelines.yml

```diff
@@ -40,12 +40,13 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Setup Java
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
-          java-version: 11
+          distribution: 'zulu'
```

Comment:
Is there a reason to choose these specifications? We recommend using jdk 19.0.1 and above in README. Please make both the mentions compatible.

Reply:
Right now even with JDK 19+ - the code actually enforces use of compatibility mode and runs under JDK 17. To be able to use proper JDK 19+ support we'll have to make some code changes - which is why I've updated it to 17 for now - once we confirm 19 support we can update toolchain as well.